config/rbac/role.yaml

```diff
@@ -1,4 +1,3 @@
-
 ---
```

Comment:
no changes in this file pls revert it

Reply:
I'm not sure how this one was updated but yeah I can revert :+1: